packages/trie/package.json

```diff
@@ -60,14 +60,14 @@
     "@types/readable-stream": "^2.3.13",
     "@types/tape": "^4.13.2",
     "0x": "^4.9.1",
-    "benchmark": "^2.1.4",
     "eslint": "^8.0.0",
     "karma": "^6.3.2",
     "karma-chrome-launcher": "^3.1.0",
     "karma-firefox-launcher": "^2.1.0",
     "karma-tap": "^4.2.0",
     "karma-typescript": "^5.5.3",
     "lmdb": "^2.5.3",
+    "micro-bmark": "^0.1.3",
```

Comment:
It would set the exact version here: `0.1.3` to improve supply chain security even more